### ISSUE TITLE
[Enhancement]: Option to "Lock" Individual Metadata Fields

### ISSUE THREAD

**Issue by chyron8472** [advplyr/audiobookshelf#928]
### Describe the feature/enhancement

I would like the option to specify, for a given library entry, which fields are not reset when I "Re-scan" it.

When I "Re-scan" an audiobook in the library, it overrides tags edited in ABS. All of my audiobooks have ID3 tagged Genres of "Audiobook." This means that any genres I previously tagged it with from within ABS are deleted, Given my feeling that genres are arbitrary depending on library size and organizational preferences, I would rather not be required to edit the genre ID3 tags directly.

**Comment by advplyr** [comment]
Do you have the setting enabled to prefer audio metadata?

**Comment by chyron8472** [comment]
I do. When new media populates, I don't want it to use the folder name as the book title.

**Comment by advplyr** [comment]
Yeah it makes sense. When you are doing a re-scan are you hoping to pick up on missing audio files or what is the goal of the re-scan?

**Comment by chyron8472** [comment]
> When you are doing a re-scan are you hoping to pick up on missing audio files

Maybe. Or perhaps to update ABS after editing some embedded tags.
